### PR TITLE
Xovis mod

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/xovis/PC2SV5/face_mask.schema.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "$id": "https://akenza.io/xovis/pc2s/face_mask.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Face mask",
+  "type": "object",
+  "properties": {
+    "faceMask": {
+      "title": "Face mask",
+      "description": "Boolean if the person has a mask on.",
```
Comment (mariusgiger) [inline types/xovis/PC2SV5/face_mask.schema.json:9]: Indicates whether a person in view wears a mask.

File: types/xovis/PC2SV5/tag.schema.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "$id": "https://akenza.io/xovis/pc2s/tag.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Tag",
+  "type": "object",
+  "properties": {
+    "tag": {
+      "title": "Tag",
+      "description": "Boolean if the person has a tag on.",
```
Comment (mariusgiger) [inline types/xovis/PC2SV5/tag.schema.json:9]: Indicates whether a person in view wears a tag.

File: types/xovis/PC2SV5/uplink spica.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
+function consume(event) {
```
Comment (mariusgiger) [inline types/xovis/PC2SV5/uplink spica.js:1]: please remove this file

File: types/xovis/PC2SV5/view_direction.schema.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://akenza.io/xovis/pc2s/view_direction.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "View direction",
+  "type": "object",
+  "properties": {
+    "xCoordinate": {
+      "title": "X Coordinate",
+      "description": "X Coordinate",
```
Comment (mariusgiger) [inline types/xovis/PC2SV5/view_direction.schema.json:9]: X coordinate of the view direction? how is the coordinate system defined?

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please change according to the comments

File: types/xovis/PC2SV5/view_direction.schema.json (hunk 4 above)
Comment (J2thatsme) [reply types/xovis/PC2SV5/view_direction.schema.json:9]: Depending on the sensor field of view. Is different from sensor to sensor.

Comment (mariusgiger) [reply types/xovis/PC2SV5/view_direction.schema.json:9]: ok but it is important to know whether the 0 is at the top or bottom left, please find that out and add the information

**Review by mariusgiger — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

File: types/xovis/PC2SV5/gender.schema.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "$id": "https://akenza.io/xovis/pc2sv5/gender.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Gender",
+  "type": "object",
+  "properties": {
+    "gender": {
```
Comment (mariusgiger) [inline types/xovis/PC2SV5/gender.schema.json:7]: how is this data intended to be used? 
what if multiple people are in the view point?

Comment (J2thatsme) [reply types/xovis/PC2SV5/gender.schema.json:7]: The sensor tracks each person as a separate entity and only assigns gender on a line crossing. Worst case its unknown.

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**